lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -6,26 +6,56 @@ class JsonApi < Base
         autoload :PaginationLinks
         autoload :FragmentCache
 
-        def initialize(serializer, options = {})
+        module ApiObjects
+          # Make JSON API top-level jsonapi member opt-in
+          # ref: http://jsonapi.org/format/#document-top-level
+          ActiveModel::Serializer.config.jsonapi_include_toplevel_member = false
+          ActiveModel::Serializer.config.jsonapi_version = '1.0'
+          ActiveModel::Serializer.config.jsonapi_toplevel_meta = {}
+          module JsonApi
```

Comment:
Agreed, as mentioned in https://github.com/rails-api/active_model_serializers/issues/1097, we should somehow namespace the config options.